### PR TITLE
Update hub-auth.adoc

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
# Description
Update documentation regarding the authentication modes of the Hub XMLRPC API, and the provided example scripts

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

### CODE REVIEW

File: modules/large-deployments/pages/hub-auth.adoc [hunk 1]
```diff
@@ -170,25 +170,27 @@ The map has two entries:
...
-HUB_LOGIN = "admin"
-HUB_PASSWORD = "admin"
 
-client = xmlrpclib.Server(<HUB_URL>, verbose=0)
+HUB_XMLRPC_API_URL = "<HUB_XMLRPC_API_URL>"
+HUB_USERNAME = "<USERNAME>"
+HUB_PASSWORD = "<PASSWORD>"
 
-hubKey = client.hub.loginWithAutoconnectMode(<HUB_LOGIN>, <HUB_PASSWORD>)
+client = xmlrpclib.Server(<HUB_XMLRPC_API_URL>, verbose=0)
```
Comment (moio) [inline modules/large-deployments/pages/hub-auth.adoc:178]:
```suggestion
client = xmlrpclib.Server(HUB_XMLRPC_API_URL, verbose=0)
```

That seems incorrect syntax, I hope my correction is correct, please test all examples manually.

File: modules/large-deployments/pages/hub-auth.adoc [hunk 2]
```diff
@@ -113,30 +113,30 @@ The map has two entries:
 import xmlrpclib
 
 
-HUB_URL = "http://localhost:8888/hub/rpc/api"
```
Comment (moio) [inline modules/large-deployments/pages/hub-auth.adoc:116]: Would it make sense to leave an example (working) URL for starters?

Comment (chiaradiamarcelo) [reply modules/large-deployments/pages/hub-auth.adoc:116]: I guess it would help, yes

File: modules/large-deployments/pages/hub-auth.adoc [hunk 3]
```diff
@@ -22,7 +22,7 @@ In manual mode, credentials have to be explicitly provided for each peripheral s
 
 A typical workflow for manual authentication is:
 
-. Credentials for the Hub are passed to the ``login`` method, and a session key for the Hub is returned (``hubKey``).
+. Credentials for the Hub are passed to the ``login`` method, and a session key for the Hub is returned (``hubSessionKey``).
 . Using the session key from the previous step, {productname} Server IDs are obtained for all the peripheral servers attached to the Hub via the ``hub.listServerIds`` method
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-auth.adoc:26]:
```suggestion
. Using the session key from the previous step, {productname} Server IDs are obtained for all the peripheral servers attached to the Hub via the ``hub.listServerIds`` method.
```

**Pull request merged**